lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -1013,6 +1013,27 @@ ValueDecl *DerivedConformance::deriveHashable(ValueDecl *requirement) {
       if (checkAndDiagnoseDisallowedContext(requirement))
         return nullptr;
 
+      // Warn the user that having a custom == almost surely require a
+      // custom Hashable conformance even if, for source stability reasons,
+      // it will be synthesized
+      auto equatableProto = C.getProtocol(KnownProtocolKind::Equatable);
+      auto eqConformance = TypeChecker::conformsToProtocol(nominalTy,
+          equatableProto, getConformanceContext());
```

Comment:
Should `getConformanceContext` be `getParentModule` instead?

Reply:
I'm pulling and building it right away (probably I will finish tomorrow knowing my MacBook Pro 😄). If this will solve it you will made me the happiest man on earth!

Reply:
It worked!